packages/css/search.css

```diff
@@ -1,49 +1,45 @@
 .ds-search {
-  --dsc-search-padding-inline-x: var(--ds-spacing-2);
-  --dsc-search-padding-inline-y: var(--ds-spacing-3);
-  --dsc-search-clear-button-size: var(--ds-sizing-6);
-  --dsc-search-input-icon: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24'%3E%3Cpath d='M10.5 3.25a7.25 7.25 0 1 0 4.57 12.88l5.41 5.41a.75.75 0 1 0 1.06-1.06l-5.41-5.41A7.25 7.25 0 0 0 10.5 3.25M4.75 10.5a5.75 5.75 0 1 1 11.5 0 5.75 5.75 0 0 1-11.5 0'/%3E%3C/svg%3E");
-  --dsc-search-clear-icon: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24'%3E%3Cpath fill='currentColor' d='M6.53 5.47a.75.75 0 0 0-1.06 1.06L10.94 12l-5.47 5.47a.75.75 0 1 0 1.06 1.06L12 13.06l5.47 5.47a.75.75 0 1 0 1.06-1.06L13.06 12l5.47-5.47a.75.75 0 0 0-1.06-1.06L12 10.94z'/%3E%3C/svg%3E");
+  --dsc-search-padding-inline: var(--ds-spacing-2);
+  --dsc-search-clear-padding: var(--ds-sizing-1);
```

Comment:
The reason I wanted to do X/Y padding/margin, is becuase it makes it easier to change with external css overriding css variables. If they change this to include two values, or more, which padding does, then our calc breaks.

Reply:
Good point, but see comment below - I think we can solve stuff with `scale` instead :D

Reply:
Scale is nice, but the problem still persists when we do `calc` on vars that *could* have two values.
I think this might be relevant for other components as well.

Reply:
Yeah, this is why we have https://github.com/digdir/designsystemet/issues/2638 ☺️